tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java

```diff
@@ -370,4 +410,6 @@ protected IStatus run(IProgressMonitor monitor) {
 		return Status.OK_STATUS;
 	}
 
+	private static String[] _searchableFolders = {"modules", "wars", "themes", "ext"};
```

Comment:
Workspaces can now have build.gradles in any folder.  (project structure is completely flexible by default now.